src/components/html/EcAccount.html

```diff
@@ -0,0 +1,29 @@
+<div class="ec-account">
+  <div v-if="isUserIdentified" class="ec-account-logged" key="account">
+    <div class="py-3">
+      <h2>{{ i18n('HelloAgain') + ' ' + nickname }}</h2>
+      <small>
+        {{ i18n('IsNotYou') + '?'}}
+        <a href="#" @click.prevent="logout" class="ec-account-logout">
+          {{ i18n('Logout') }}
+        </a>
+      </small>
+    </div>
+    <b-tabs content-class="mt-3">
+      <b-tab :title="i18n('Registration')" active>
+        <ec-account-form :customer.sync="localCustomer"/>
+      </b-tab>
+      <b-tab :title="i18n('Addresses')">
+        <ec-addresses :customer.sync="localCustomer"/>
+      </b-tab>
+      <b-tab :title="i18n('Orders')">
+        <ec-orders-list :ecomPassport.sync="ecomPassport"></ec-orders-list>
+      </b-tab>
+    </b-tabs>
+  </div>
+
+  <div key="login" v-else>
+    <ec-identify @login="login" @update="isUserIdentified = true" :customerEmail.sync="customerEmail" />
```

Comment:
como está você está permitindo que o usuário acesse o account mesmo sem ter conta...
```suggestion
    <ec-identify @login="login" :customerEmail.sync="customerEmail" />
```

Reply:
lembra que este `isUserIdentified` do checkout tinha o intuito só de saber se sabia o email do usuário, se há uma identificação...

Reply:
mas no caso do account isto não é o suficiente, você precisa saber se ele está logado